app/controllers/devise_remote/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module DeviseRemote
+  class SessionsController < ApplicationController
```

Comment:
Should this be a module so you can easily override the before_destroy?

Reply:
@cam156 I thought about that, but decided the `prepend` approach might be better. What do you think? Either way, I should add some documentation.

Reply:
I keep forgetting about prepends!

Reply:
@cam156 I know! So do I!!! Old habits..